lib/functions.php

```diff
@@ -414,7 +414,7 @@ function elgg_solr_get_adapter_options() {
 function elgg_solr_has_settings() {
 	$host = elgg_get_plugin_setting('host', 'elgg_solr');
 	$port = elgg_get_plugin_setting('port', 'elgg_solr');
-	$path = elgg_get_plugin_setting('path', 'elgg_solr');
+	$path = elgg_get_plugin_setting('solr_path', 'elgg_solr');
```

Comment:
nevermind, I see, thought you were changing it away from what it was supposed to be